tests/voq/test_voq_disrupts.py

```diff
@@ -143,7 +143,7 @@ def check_ip_fwd(duthosts, all_cfg_facts, nbrhosts, tbinfo):
                              ttl=ttl, ttl_change=0)
 
                 vm_host_to_A = nbrhosts[ports['portA']['nbr_vm']]['host']
-                check_packet(eos_ping, ports, 'portA', 'portA', dst_ip_fld='my_lb_ip', src_ip_fld='nbr_lb',
+                check_packet(eos_ping, ports, 'portA', 'portA', dst_ip_fld='my_lb4096_ip', src_ip_fld='nbr_lb',
```

Comment:
Check more on this, the loopback4096 is not advertised to the peers outside the chassis. 
For the T1 neighbors the ping to any LC's `loopback4096`  will work because it will use the default route. For T3 neighbors the ping to any LC's `loopback4096` is not expected to work. In this test it seems `portA` is always a port for T1 neighbor

Reply:
This will work fine in our testbed due to default route in T1 points to th eonly T2 which is out DUT, but since this is not the real production scenario ( production scenario is to make sure we are able to ping the loopback IP which is unique to LC ) -- I feel we should leave this test case as it is now -- without this change, 

Please share comments

Reply:
Yes Judy, it works with the default route. This test is basically per asic level for T2 chassis and to be absolutely sure that we get the respective reply from the corresponding asic, this loopback4096 IP helps. With only loopback IP and if we get a reply from other asic on the chassis, this test doesn't serve the required purpose.